recipes/prgenv-nvidia/24.6/gh200/environments.yaml

```diff
@@ -0,0 +1,32 @@
+nvidia-env:
+  compiler:
+      - toolchain: gcc
+        spec: gcc
+      - toolchain: llvm
+        spec: nvhpc
+  mpi:
+      spec: cray-mpich@8.1.29%nvhpc
+      gpu: cuda
+  unify: true
+  specs:
+  - aws-ofi-nccl@master
+  - cmake
+  - cuda@12.4
+  - fftw
+  - fmt
+  - hdf5%nvhpc +fortran
+  - libtree
+  - meson
+  - nccl
+  - nccl-tests
+  - ninja
+  - openblas threads=openmp
```

Comment:
openblas is linked to `libgomp.so` isn't this creating a conflict with `nvomp.so`? Ie two different openmp providers being used?  Same for fftw afaik.